nodejs/json-rpc-relay/integration.tests/contracts/deploy.contract.test.js

```diff
@@ -0,0 +1,34 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const fs = require('fs');
+const path = require('path');
+const { ethers } = require("ethers");
+const urlHandler = require('../../../handlers/url.handler');
+const privateECDSAAccount = `0x8eaecc9af6db21e7bc66032d9891563e68e2930ac2a09da6c5872a92a3790203`;
+
+
+describe('[Ethers] Deploy Contract ', () => {
+
+    test('Testnet | Should able to Deploy Contract', async () => {
```

Comment:
we should add more tests later for same contract like `getContractBalance()`

Reply:
for sure , wanna add a writing method as well , deposit hbars and so on, will keep working on this